dsp-shared/src/main/scala/dsp/valueobjects/LangString.scala

```diff
@@ -0,0 +1,49 @@
+package dsp.valueobjects
+
+import dsp.errors.ValidationException
+import zio.prelude.Validation
+
+sealed abstract case class LangString private (language: LanguageCode, value: String)
+
+object LangString {
+  def make(language: LanguageCode, value: String): Validation[ValidationException, LangString] =
+    if (value.isBlank()) {
+      Validation.fail(ValidationException(LangStringErrorMessages.LangStringValueEmpty))
+    } else {
+      Validation.succeed(new LangString(language, value) {})
+    }
+
+  def make(language: String, value: String): Validation[ValidationException, LangString] =
```

Comment:
Now that we have value objects and make methods, it would be good to test them with unit tests (for all value objects you created in this PR).

Reply:
done